src/Android/MainActivity.cs

```diff
@@ -401,5 +404,12 @@ private async Task StopEventAlarmAsync()
             alarmManager.Cancel(_eventUploadPendingIntent);
             await _eventService.UploadEventsAsync();
         }
+
+        private void DisableAndroidFontScale()
+        {
+            //As we are using NamedSizes the xamarin will change the font size. So we are disabling the Android scaling.
+            Resources.Configuration.FontScale = 1f;
+            BaseContext.Resources.DisplayMetrics.ScaledDensity = Resources.Configuration.FontScale * (float)DeviceDisplay.MainDisplayInfo.Density;
+        }
```

Comment:
Not 100% sure if this is safe enough that doesn't throw exception at some case or device specifics. Just in case, could you wrap this in a try...catch that logs the exception to AppCenter? So that if something happens the app doesn't crash and the users can use the app nonetheless given that you're calling it from `OnCreate`